tests/filters_test.rs

```diff
@@ -267,6 +296,16 @@ fn can_convert_table_to_tsv_text_and_from_tsv_text_back_into_table() {
     assert_eq!(actual, "SPAIN");
 }
 
+#[test]
+fn can_convert_table_to_tsv_text_and_from_tsv_text_back_into_table_using_csv_separator() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats",
+        r"open caco3_plastics.csv | to-tsv | from-csv --separator '\t' | first 1 | get origin | echo $it"
```

Comment:
Shouldn't this be caco3_plastics.tsv ?

Reply:
Nevermind :)

Reply:
ah yes! it should, good spot

Reply:
let me just force push over this, if that's ok with you?

Reply:
sure, thanks!

Reply:
Although I think this test is correct. You are opening a csv file and getting a table, converting to tsv and using the expected `from-csv` with the separator wanted in this case.

This is correct.

Reply:
it's slightly confusing... but yes, it wouldn't matter what format the file was as long as it reads as a table, then we make tsv text (so we _add_ tabs) and then we parse that as csv separated by tabs...

I think this does the job of demonstrating that `tsv` and `csv --separator '\t'` are interoperable :D